### PR TITLE
Engineering Belt - 10 Items, Medical Belt - 12 Items, Cable Coil - Max 40 now.

### PR DESCRIPTION
**Pull request by JamieD1**
# Github documenting your Pull Request

Makes Max items 10 - Makes Cable coil weight change based on amount, added 10 to max coil.

Medical to 12 items.

Closes: #12099

# Wiki Documentation

Makes Max items 10 - Makes Cable coil weight change based on amount, added 10 to max coil.

Medical to 12 items.

# Changelog

:cl:  
tweak: Makes Max items 10 for engi belt - Makes Cable coil weight change based on amount, added 10 to max coil. Medical to 12
/:cl:

### CODE REVIEW

**Comment by N3D6** [comment]
good compromise i suppose

**Comment by Marmio64** [comment]
this is pretty dumb and annoying when putting cable coils into a toolbox since it doesn't let you put in 40 cable coil in one stack but it lets you split in half and put inside, and then they both merge into one stack anyways?

File: code/modules/power/cable.dm [hunk 1]
```diff
@@ -481,15 +481,15 @@ GLOBAL_LIST_INIT(cable_coil_recipes, list (new/datum/stack_recipe("cable restrai
 	item_color = "red"
 	desc = "A coil of insulated power cable."
 	throwforce = 0
-	w_class = WEIGHT_CLASS_SMALL
+	w_class = WEIGHT_CLASS_TINY
```
Comment (JamieD1) [inline code/modules/power/cable.dm]:
```suggestion
	w_class = WEIGHT_CLASS_SMALL
```

File: code/modules/power/cable.dm [hunk 2]
```diff
@@ -481,15 +481,15 @@ GLOBAL_LIST_INIT(cable_coil_recipes, list (new/datum/stack_recipe("cable restrai
...
 	throw_speed = 3
 	throw_range = 5
 	materials = list(/datum/material/iron=10, /datum/material/glass=5)
 	flags_1 = CONDUCT_1
 	slot_flags = ITEM_SLOT_BELT
 	attack_verb = list("whipped", "lashed", "disciplined", "flogged")
 	singular_name = "cable piece"
-	full_w_class = WEIGHT_CLASS_SMALL
+	full_w_class = WEIGHT_CLASS_NORMAL
```
Comment (JamieD1) [inline code/modules/power/cable.dm]:
```suggestion
	full_w_class = WEIGHT_CLASS_SMALL

**Pull request merged**